### DISCUSSION TITLE
Milvus Binaries

### DISCUSSION THREAD

**Discussion by zirota** [milvus-io/milvus#25210] [Q&A and General discussion]
I am trying to deploy milvus in a corporate air-gapped environment and requires Milvus to be built on a custom RHEL8 image

However, there doesn't seem to be any binaries for Milvus available to download for 2.2.X for me to use in my image building.

Are there plans to release any binaries so that this can be done easily?

**Comment by yhmo** [comment]
The Milvus binaries in the official docker image are built on CentOS, I am not sure it can work on RHEL8. You can copy the binaries from a Milvus container, under the path /milvus/bin and /milvus/lib

**Reply by zirota** [reply]
I tried this as well, but ran into an issue which says:
 `/lib64/libm.so.6: version 'GLIBC_2.29' not found (required by /milvus/lib/libjemalloc.so)`
 
 Does Milvus require this particular version of glibc?

**Reply by yhmo** [reply]
I guess your gcc version is not >= 9. 
Use `docker exec` go into the milvus container, you can use "ldd" command to check the dependencies of milvus. The gcc version of the container is v9.4.
